stable/jenkins/templates/jenkins-master-ingress.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.Master.HostName }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "fullname" . }}
```

Comment:
The nginx controller requires at least a `kubernetes.io/ingress.class: nginx` annotation on the Ingress resource, but supports others as well. Perhaps provide an `ingressAnnotations` value ?

values.yaml:
```
...
ingressAnnotations:
  # kubernetes.io/ingress.class: nginx
  # kubernetes.io/tls-acme: "true"
...
```

jenkins-master-ingress.yaml:
```
...
metadata:
{{- if .Values.ingressAnnotations }}
  annotations:
{{ toYaml .Values.ingressAnnotations | indent 4 }}
{{- end }}
...
```